src/sentry/integrations/github/client.py

```diff
@@ -229,17 +229,23 @@ def get_pullrequest_from_commit(self, repo: str, sha: str) -> JSONData:
 
         Returns the merged pull request that introduced the commit to the repository. If the commit is not present in the default branch, will only return open pull requests associated with the commit.
         """
-        pullrequest: JSONData = self.get(f"/repos/{repo}/commits/{sha}/pulls")
-        return pullrequest
+        return self.get(f"/repos/{repo}/commits/{sha}/pulls")
 
-    def get_pullrequest(self, repo: str, pull_number: int) -> JSONData:
+    def get_pullrequest(self, repo: str, pull_number: str) -> JSONData:
```

Comment:
pull number is a string? 🤔 🤷

Reply:
it's stored as "key" in the db which is a string too 🤷🏻‍♀️